apps/web/src/components/field/index.tsx

```diff
@@ -203,32 +204,36 @@ function Field(props: FieldProps) {
             }}
           >
             {rightActions.map((action) => (
-              <Button
-                id={action.id}
-                key={action.testId}
-                type="button"
-                variant={"secondary"}
-                data-test-id={action.testId}
-                onClick={action.onClick}
-                sx={{
-                  p: 0,
-                  px: 1,
-                  height: "100%",
-                  bg: "transparent",
-                  borderRadius: 0,
-                  margin: 0,
-                  display: "flex",
-                  alignItems: "center",
-                  justifyContent: "center"
-                }}
-                disabled={action.disabled}
-              >
-                {action.component ? (
-                  action.component
-                ) : action.icon ? (
-                  <action.icon size={16} />
-                ) : null}
-              </Button>
+              <>
+                {action.hidden ? null : (
```

Comment:
Maybe we can filter out the `rightActions` instead of a condition here. Also this should be handled in `leftActions` too.